### ISSUE TITLE
Some questions about the data set

### ISSUE THREAD

**Issue by Bin-ze** [hu64/SpotNet#17]
Sorry to disturb you.
Recently, I am doing some research on the UAV data set, but I found that I cannot find a suitable way to verify the results of my network. The official data does not seem to provide a test set, and it can be verified without submitting it online. Performance online test, so I want to ask how you used the data set in the first place

**Comment by hu64** [comment]
Hi! To my knowledge, there is an official test set as well as a Matlab evaluation toolkit provided by the authors to evaluate the results. The data contains a lot of ignore regions defined in this toolkit.  This is how I produced the final results. To train, I divided the training set into a train and validation split, without using any of the test images. Although I must tell you it's a bit weird, the website where I downloaded the images no longer exists: sites.google.com/site/daviddo0323/projects/uavdt. I cannot say if they are still available somewhere.

**Comment by Bin-ze** [comment]
I want to process the data set into COCO or VOC format, but after training in this way, how can I ensure that the comparison with the results in other papers is fair, because the test set is randomly divided by itself.

**Comment by hu64** [comment]
I think to be fair with the results of other papers, you really have to evaluate in the same settings with the same tools. You can always train with other formats of data and then reconvert in theirs to evaluate with their tool. I mean, you can always produce results of other methods yourself in your own settings, but in my experience that isn't seen so well with reviewers.

**Comment by Bin-ze** [comment]
I see what you mean. I have actually downloaded the evaluation tool of the official data set, but I didn't find the test set. If so, how can I ensure comparison with the results in other papers? I have read your paper carefully. Can you send me a copy of the evaluation tool you wrote your previous paper? I want to evaluate my improvement on this basis. I also use centernet, but I have made new improvements. I want to verify the gap between me and sota on this dataset.
发自我的iPhone

**Comment by Bin-ze** [comment]
I checked your new paper, FFAVOD, and I found that you used the verification set in the code, and generated the result in run_on_csv_method.py. Is the data set you use divided by yourself?
Sorry to interrupt you, because your article is SOTA on this data set, I would like to ask you, if you can send me a copy of your compiled data set, I will be very grateful.
Because it is difficult for me to operate from the official data set to achieve the effect I want.
Regarding the evaluation indicators, are the indicators on the UAVDT data set the AP indicators of coco? I am using AP indicators to evaluate my model, and the news from the Internet is not very accurate. I learned from your I saw your introduction to the indicator in the spot net paper, which seems to be different from the coco indicator? Looking forward to your reply!

**Comment by hu64** [comment]
For your first request, I am sorry, but I no longer have access to the dataset. Although I can tell you, I did not decide myself of the test set, It is defined in the Matlab evaluation tool. The only I did myself is decide of a small subset of the train set for validation purposes. To get access to the dataset and evaluation tool, may I suggest that you contact the authors of the dataset? For your second question, I can tell you that the AP calculated is not the same as for COCO. It is something like AP 0.7 rather than AP 05:0.95, you see what I mean. And there are also a lot of ignore regions which you have to take into account.

**Comment by hu64** [comment]
Hi, since I answered your question there: [https://github.com/hu64/FFAVOD/issues/2](https://github.com/hu64/FFAVOD/issues/2) I will close this issue. Good luck with your research!
